### ISSUE TITLE
Feature request: Flag for custom IP for "dsnet add"

### ISSUE THREAD

**Issue by haary** [naggie/dsnet#89]
Sure, one can edit /etc/dsnetconfig.json, set a custom IP address and run "dsnet sync". But it would be nice to have the ability to config a custom IP address when adding new peers. Because auto-assigning the lowest available IP address is not always wanted. 

E.g. something like  `dsnet add myhost --ipv4 10.80.54.100` would be nice.

**Comment by naggie** [comment]
Yep, good idea. I'd be happy to merge a PR for that + an IPv6 flag
